index.js

```diff
@@ -104,7 +104,13 @@ const performAction = async (action) => {
         process.exit(1)
     }
     try {
-        return request(action.method, action.url).send(action.json_body)
+        var r = request(action.method, action.url)
+        if (action.headers != undefined) {
```

Comment:
I think it would be nicer to avoid the if statement and only have one code path. Something like
```
const r = request(action.method, action.url);
Object.entries(action.headers || {}).forEach(([key, value]) => {
    r.set(key, value);
});
return r.send(action.json_body);
```